.wordlist.txt

```diff
@@ -347,6 +352,8 @@ TPU
 TPUs
 TRL
 TTFT
+TTGIR
```

Comment:
In the context of Triton, which is an open-source machine learning compiler developed by OpenAI, TTGIR stands for Triton Tensor Graph Intermediate Representation